uint16 ledIndex
uint16 red
uint16 green
uint16 blue
float32 intensity
---
